### PR TITLE
Alumni Section Update

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

File: data/data-home.json [hunk 1]
```diff
@@ -150,6 +144,12 @@
             "linkedin":"https://www.linkedin.com/in/hanutsingh",
             "picurl":"images/alumini/default.jpeg"
           },
+          {
+            "name":"SHIVAM RAJPUT",
```
Comment (thisisayush) [inline data/data-home.json]: Change from UPPERCASE to Sentence Case (if that's the word for it)

Comment (vipulgupta2048) [reply data/data-home.json]: Was just following the format everything else was also capital.

**Comment by thisisayush** [comment]
Looks good to me.

**Comment by vipulgupta2048** [comment]
Done. Please call me when you are merging this. Gotta see for my self what Travis is doing or actually not doing after PR merge.

**Comment by thisisayush** [comment]
Either remove the changes from index.html or do not wait for Travis.

**Comment by vipulgupta2048** [comment]
Oh, I didn't see I already built it. Force of habit, I guess.
Going with the former option.

**Comment by vipulgupta2048** [comment]
@thisisayush REAL LIFE EXAMPLE OF WHY I DON'T RECOMMEND COMMITTING directly to master. I am not correcting my history because of you.

**Comment by vipulgupta2048** [comment]
Ready to go, but as you can see. Travis is building now, and deploying as well. 
From the latest commit https://github.com/asetalias/asetalias.github.io/commit/c1251e0d6c1545d1f64373976e75af5089d80b47

**Comment by thisisayush** [comment]
There are only 2 logical commits in this. Squash the rest?

**Comment by vipulgupta2048** [comment]
Man, I may as well open a box of spiders than try doing anything to the history of the branch. Just merge this, I will edit the history for asetalias later :imp:

**Comment by vipulgupta2048** [comment]
Just kidding

**Comment by thisisayush** [comment]
Keep playing. I'm not merging it unless you fix this. 😂

**Comment by vipulgupta2048** [comment]
@thisisayush This has literally a good number of commits. This is fine, merge it or check discussion in why we can because of your 8 commits that were force pushed into master.

**Comment by vipulgupta2048** [comment]
@thisisayush This didn't work somehow??

**Pull request merged**